core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210615_sharded_entity_tables.xml

```diff
@@ -3,38 +3,33 @@
 
     <!-- create db functions and stored procedures that will help with the db migration -->
     <changeSet logicalFilePath="dummy" author="davidan" id="create_shard_functions_and_procedures">
-        <sql stripComments="true" endDelimiter="/">
```

Comment:
Out of curiosity, how are you testing this changeset works? Are you cloning the dev db, or running it on a local db?

Reply:
Unit tests prove it's not entirely broken - but then I ran it against a clone of the dev db and verified that shards contain data and I can query against that data.